core/domain/topic_services.py

```diff
@@ -411,6 +415,10 @@ def _save_topic(committer_id, topic, commit_message, change_list):
     topic.validate(strict=topic_rights.topic_is_published)
 
     topic_model = topic_models.TopicModel.get(topic.id, strict=False)
+    if not topic.are_subtopic_url_fragments_unique():
```

Comment:
Shouldn't this be added to `topic.validate`?

Reply:
I'll address this in #10041 (which builds on top of this PR). Is that OK?

Reply:
Yup -- please file an issue to track and have #10041 (among other things) fix the issue.

Reply:
(Also, link the issue here once you've filed it. This is standard practice for any deferred changes so please feel free to go ahead and do that proactively next time -- in the worst case, where the deferral isn't acceptable, we would just close it.)

Reply:
Noted. Created #10191 to track.